Pachinko/Pachinko.c

```diff
@@ -0,0 +1,104 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <time.h>
+
+#define MINIMUM (100)
+#define LINESIZE (1024)
+
+/* Chooses a random number between 0 and 4 inclusively */
+void pickRandom(int* number)
+{
+	*number = rand() % 4;
+}
+
+/* Spins the slots */
+void playSlots(int slot[], size_t n)
+{
+	size_t i;
+	for (i = 0; i < 3; i++)
+	{
+		pickRandom(&slot[i]);
+	}
+}
+
+/* Gets the amount the player wants to bet and stores it into bet */
+int insertCoin(int* bet)
+{
+	char line[LINESIZE];
+	*bet = 0;
+	while (1)
+	{
+		printf("How much would you like to bet?");
+		if (!fgets(line, LINESIZE, stdin))
+		{
+			clearerr(stdin);
+			printf("Error reading line");
+			return 1;
+		}
+		if (sscanf_s(line, "%d", bet) == 1 && *bet >= 100)
+		{
+			return 0;
+		}
+		printf("%s\n", "The minimum to bet is 100 Credits, please pick a larger number");
+	}
+}
+
+/* Checks if the slots match and pays the player accordingly */
+void payOut(int slot[], int* credits, int* bet)
+{
+	size_t i;
+	int* p = &slot[0];
+	for (i = 0; i < 3; i++)
+	{
+		switch (slot[i])
+		{
+		case 0: printf("%s", "@");
```

Comment:
Use the following format
switch(slot[i])
case 0:
{
    printf("%s", "@");
    break;
}
case 1:
...